src/Core/Migration/V6_4/Migration1670090989AddIndexOrderOrderNumber.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_4;
```

Comment:
we are in the process of releasing the 6.5 major version, so trunk branch is already for 6.5 changes only, therefore you should put the migration in the `V6_5` namespace

Reply:
Will do